### ISSUE TITLE
Release 2.2

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/project#30]

**Comment by cmoulliard** [comment]
Why the issue `Add OpenShift labels to Snowdrop examples` has been removed from this epic ? @iocanel

**Comment by cmoulliard** [comment]
Why the issue `Upgrade Spring Boot to 2.2.5.RELEASE` appears twice ? @gytis

**Comment by gytis** [comment]
Their from different organisations. One from Narayana and another from Vert.x

**Comment by metacosm** [comment]
Why is this issue under snowdrop/project and not under the appropriate
github project?

**Comment by gytis** [comment]
@metacosm I agree that it should be in `spring-boot-bom` project. But I think let's keep it here for this release, because it already has a lot of dependencies linked.
